.travis.yml

```diff
@@ -15,9 +15,6 @@ php:
   - hhvm
 
 matrix:
-  include:
-    - php: 5.3
-      env: deps=low
```

Comment:
One issue is that you try to mock interfaces that do not exist in older Symfony versions. You have to check for their existence first and then either mock them or the old security context.